server/templates/custom_dc/unsdg/base.html

```diff
@@ -0,0 +1,109 @@
+{#-
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-#}
+{#-
+Required variables:
+title: page title
+main_id: DOM Element ID for the main content block
+page_id: DOM Element ID for the html element
+
+Optional variables:
+is_hide_full_footer: boolean, if true, hides the full expanded footer. Default false
+is_hide_sub_footer: boolean, if true, hides the sub footer. Default false
+subpage_title: string, if specified, will be displayed in the header next to Data Commons
+locale: string, value for html lang attr
+
+Blocks to override:
+head - additional head elements
+content - main page content
+-#}
+<!DOCTYPE html>
+
+<html lang={{ locale }} id={{ page_id }}>
+
+<head>
+  {# Enable Google Analytics with cookieless tracking. #}
+  {% if GA_ACCOUNT %}
+  <script async src="https://www.googletagmanager.com/gtag/js?id={{ GA_ACCOUNT }}"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+    gtag('config', '{{ GA_ACCOUNT }}', { client_storage: 'none', anonymize_ip: true });
+  </script>
+  {% endif %}
+  <title>{{ title }} - UN SDG Data Commons</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="icon" href="/favicon.ico" type="image/x-icon">
+  <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@300;600&family=Roboto&display=swap"
+    rel="stylesheet">
+  {% block head %}
+  {% endblock %}
+  <link href="/custom_dc/unsdg/css/custom.css" rel="stylesheet">
+</head>
+
+<body id="dc-fa">
```

Comment:
nit: id="dc-unsdg" or just removed if not used

Reply:
Done.